opensciencegrid/ospool-cm/opt/ospool/update-prios

```diff
@@ -0,0 +1,27 @@
+#!/bin/python3
+
+import re
+import htcondor
+from pprint import pprint
+
+
+def update_user(ad):
+    current_factor = int(ad["PriorityFactor"])
+    new_factor = current_factor
+
+    # ap20/21 migration
```

Comment:
This could use some clarification: what's the goal of setting the factor to 500 for login04/05 users? To my untrained eye, it's an arbitrary number